static/themes/dark.css

```diff
@@ -0,0 +1,76 @@
+::selection {
+  background-color: #004daa !important;
```

Comment:
There are too many `!important` here.
It should not be necessary. We should not have a `dark.css` overridden the `app.css` styles.

Instead, we should have the styles defined on `light.css`, `dark.css`, and on `app.css` only the common styles for both themes that don't depend on the color schema (for instance, the font sizes).
Then, we won't need to use `!important`.